test/test.rb

```diff
@@ -5,64 +5,64 @@
+      assert_equal test_case[0, 10].mean.round(6), test_case[11].round(6)
     end
   end
 
   def test_median
     @data.each do |test_case|
-      assert_equal test_case[0,10].median.round(6), test_case[12].round(6)
+      assert_equal test_case[0, 10].median.round(6), test_case[12].round(6)
     end
   end
 
   def test_variance
     @data.each do |test_case|
-      assert_equal test_case[0,10].variance.round(6), test_case[13].round(6)
+      assert_equal test_case[0, 10].variance.round(6), test_case[13].round(6)
     end
   end
 
   def test_standard_deviation
     @data.each do |test_case|
-      assert_equal test_case[0,10].standard_deviation.round(6), test_case[14].round(6)
+      assert_equal test_case[0, 10].standard_deviation.round(6), test_case[14].round(6)
     end
   end
 
   def test_percentile
     @data.each do |test_case|
-      assert_equal test_case[0,10].percentile(0).round(6), test_case[15].round(6)
-      assert_equal test_case[0,10].percentile(10).round(6), test_case[16].round(6)
-      assert_equal test_case[0,10].percentile(20).round(6), test_case[17].round(6)
-      assert_equal test_case[0,10].percentile(30).round(6), test_case[18].round(6)
-      assert_equal test_case[0,10].percentile(40).round(6), test_case[19].round(6)
-      assert_equal test_case[0,10].percentile(50).round(6), test_case[20].round(6)
-      assert_equal test_case[0,10].percentile(60).round(6), test_case[21].round(6)
-      assert_equal test_case[0,10].percentile(70).round(6), test_case[22].round(6)
-      assert_equal test_case[0,10].percentile(80).round(6), test_case[23].round(6)
-      assert_equal test_case[0,10].percentile(90).round(6), test_case[24].round(6)
-      assert_equal test_case[0,10].percentile(100).round(6), test_case[25].round(6)
+      assert_equal test_case[0, 10].percentile(0).round(6), test_case[15].round(6)
+      assert_equal test_case[0, 10].percentile(10).round(6), test_case[16].round(6)
+      assert_equal test_case[0, 10].percentile(20).round(6), test_case[17].round(6)
+      assert_equal test_case[0, 10].percentile(30).round(6), test_case[18].round(6)
+      assert_equal test_case[0, 10].percentile(40).round(6), test_case[19].round(6)
+      assert_equal test_case[0, 10].percentile(50).round(6), test_case[20].round(6)
+      assert_equal test_case[0, 10].percentile(60).round(6), test_case[21].round(6)
+      assert_equal test_case[0, 10].percentile(70).round(6), test_case[22].round(6)
+      assert_equal test_case[0, 10].percentile(80).round(6), test_case[23].round(6)
+      assert_equal test_case[0, 10].percentile(90).round(6), test_case[24].round(6)
+      assert_equal test_case[0, 10].percentile(100).round(6), test_case[25].round(6)
     end
   end
 
   def test_percentile_rank
     @data.each do |test_case|
-      assert_equal test_case[0,10].percentile_rank(50).round(6), test_case[26].round(6)
+      assert_equal test_case[0, 10].percentile_rank(50).round(6), test_case[26].round(6)
```

Comment:
I would love to see rubocop implemented in this repository. Would you be open to moving any style related changes to a new pull request, one which included the rubocop package and a configuration?

Reply:
Sure, I will do that any preference on the rubocop style guide or just the default one, one thing I would suggest is to increase the line size from 80 columns to something bigger specially for the test parts.

Reply:
Hey, I opened the Rubocop PR as you asked, after merge we can work in the style changes 👓